### DISCUSSION TITLE
How to install jbig2dec on Windows?

### DISCUSSION THREAD

**Discussion by hanakp-VUT** [ocrmypdf/OCRmyPDF#1357] [Q&A]
Hello, I'm normally using OCRmyPDF on Linux, but today I tried it on Windows. I downloaded and installed 3 packages (Python, Tesseract and Ghostscript) as explained in [the docs](https://ocrmypdf.readthedocs.io/en/latest/installation.html#installing-on-windows), although I did that manually, i.e. without Winget. Then I installed OCRmyPDF via pip3. It worked, but reported that Unpaper package is missing, I had to add it with Chocolatey. Now there is one last problem I'm unable to solve: OCRmyPDF produces this warning whenver I run it:

`UserWarning: pikepdf is missing some specialized decoders (probably JBIG2) so not all stream contents can be tested. `

In [this thread](https://github.com/ocrmypdf/ocrmypdf/issues/1225), the author mentions it could be solved by installing jbig2dec, which is now hosted [elsewhere on Github](https://github.com/ArtifexSoftware/jbig2dec). But... how do I actually do that on Windows? I'd have to compile it? Or have I encountered these issues merely because I haven't used Winget as was recommended?

**Comment by jbarlow83** [comment]
The warning is harmless and can be ignored in most cases. It seems that the vendor is no longer releasing Windows builds of jbig2dec. I'll suppress the warning in a future release.
